src/js/components/LongDashPlaceholder.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Component introduced so we don't need to hardcode Unicode all over the place.
This is also an approach towards component based UI.

Why not a more general naming? because as per component based UIs the components are disposable and whenever something new is needed we create a new component instead of extending or change. Until no more is in use that way we can just delete.